conf/main/mindmaps-engine.properties

```diff
@@ -47,3 +47,6 @@ backgroundTasks.time-lapse=60000
 
 #Loader Config
 loader.threads=0
+
+#MindmapsDB version
+project.version=0.1.0-SNAPSHOT
```

Comment:
Graql shell has a way to get the version straight from maven! Look in java-templates